### DISCUSSION TITLE
Alerters: support wechat.

### DISCUSSION THREAD

**Discussion by daiwei233** [jertel/elastalert2#280] [Ideas]
Wechat is a Chinese enterprise IM, offical website: https://open.work.weixin.qq.com/api/doc/90000/90135/90664.

Just like the alertmanager: https://prometheus.io/docs/alerting/latest/configuration/#wechat_config

**Comment by nsano-rururu** [comment]
We are waiting for your pull request.

**Reply by nsano-rururu** [reply]
https://open.work.weixin.qq.com/api/doc/90000/90136/91770

**Reply by nsano-rururu** [reply]
I don't use wechat so I can't check the operation.
Please refer to the following to make a pull request.

https://github.com/Zhang21/elastalert-wechatrobot-plugin

**Reply by daiwei233** [reply]
okay.

**Reply by nsano-rururu** [reply]
The files to modify are loaders.py and schema.yaml. 
The files to add are wechat.py and wechat_test.py. 
Please note that the source code structure is different from yelp elastalert.

```
elastalert
  loaders.py
  schema.yaml
  /alerters
    wechat.py
  /tests
    /alerters
      wechat_test.py
```

**Reply by nsano-rururu** [reply]
Added alert name to elastalert.rst. Added description about setting wechat to ruletypes.rst.

```
docs/source/
  elastalert.rst
  ruletypes.rst
```

**Reply by daiwei233** [reply]
Hi, i have made a pull request: https://github.com/jertel/elastalert2/pull/355
